Bang/Players/Player.cs

```diff
@@ -145,7 +150,10 @@ public void RegainLifePoint()
             PlayerTablet.Health++;
         }
 
-        public void LoseLifePoint(int loseLifeAmount = 1)
+        public void LoseLifePoint(int loseLifeAmount = 1) => LoseLifePoint(null, loseLifeAmount);
+        
+        
+        public void LoseLifePoint(Player responsible, int loseLifeAmount)
```

Comment:
**minor:** We almost everywhere call LoseLifePoint() method with loseLifeAmount parameter equals 1. Maybe we can set it as default parameter

Reply:
Fixed